docs/manual/scala/guide/cluster/UsingAkkaPersistenceTyped.md

```diff
@@ -150,19 +148,21 @@ To locate the correct actor across the cluster, you need to specify the `EntityT
 
 The `EntityRef` is similar to an `ActorRef` but denotes the actor is sharded. Interacting with an `EntityRef` implies the messages exchanged with the actor may need to travel over the wire to another node.
 
-#### Considerations on using `ask` pattern
+#### Considerations on using the ask pattern
```

Comment:
This is to avoid this: 

![Screenshot 2019-11-07 at 13 52 25](https://user-images.githubusercontent.com/502982/68392764-cd416e80-016a-11ea-99b9-5ce56cfbee70.png)